### PR TITLE
[Order] Change Adjustment description to label

### PR DESCRIPTION
**Pull request by Zales0123**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | https://github.com/Sylius/Sylius/issues/3849
| License       | MIT
| Doc PR        | https://github.com/Sylius/Sylius-Docs/pull/396

### CODE REVIEW

File: src/Sylius/Component/Core/OrderProcessing/ShippingChargesProcessor.php [hunk 1]
```diff
@@ -63,11 +63,12 @@ public function applyShippingCharges(OrderInterface $order)
             $adjustment = $this->adjustmentFactory->createNew();
             $adjustment->setType(AdjustmentInterface::SHIPPING_ADJUSTMENT);
             $adjustment->setAmount($shippingCharge);
-            $adjustment->setDescription($shipment->getMethod()->getName());
+            $adjustment->setLabel($shipment->getMethod()->getName());
 
             $order->addAdjustment($adjustment);
         }
 
+
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Core/OrderProcessing/ShippingChargesProcessor.php]: Should be reverted.

**Comment by peteward** [comment]
:+1:

**Comment by mmckelviereiss** [comment]
:+1:

**Comment by Zales0123** [comment]
@pjedrzejewski @michalmarcinkowski same here, Scrutinizer error :(

**Comment by pjedrzejewski** [comment]
@Zales0123 Please update UPGRADE file.

**Comment by Zales0123** [comment]
@pjedrzejewski done :+1:

File: src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadPromotionsData.php [hunk 2]
```diff
@@ -121,18 +121,18 @@ protected function createAction($type, array $configuration)
...
      * @return PromotionInterface
      */
-    protected function createPromotion($code, $name, $description, array $rules, array $actions)
+    protected function createPromotion($code, $name, $label, array $rules, array $actions)
     {
         /** @var $promotion PromotionInterface */
         $promotion = $this->getPromotionFactory()->createNew();
         $promotion->setName($name);
-        $promotion->setDescription($description);
+        $promotion->setDescription($label);
```
Comment (pamil) [inline src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadPromotionsData.php]: `setDescription($label)`?

Comment (Zales0123) [inline src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadPromotionsData.php]: Good point :smile:

**Comment by pjedrzejewski** [comment]
Thanks Mateusz!

**Pull request merged**